content_scripts/inject_script.coffee

```diff
@@ -0,0 +1,16 @@
+injectScripts = ["pages/addEventListener_hook.js"]
+
+fetchFileContents = (extensionFileName) ->
+  req = new XMLHttpRequest()
+  req.open("GET", chrome.runtime.getURL(extensionFileName), false) # false => synchronous
+  req.send()
+  req.responseText
+
+for script in injectScripts
+  # Inject the script, which will be executed before the page scripts, as long as it is injected directly as
+  # text.
+  # TODO(mrmr1993): Find a reasonable way to inline the scripts to inject here at build time, since doing a
+  # synchronous XMLHttpRequest will slow down every page load.
+  scriptEl = document.createElement "script"
+  scriptEl.innerHTML = fetchFileContents script
+  document.documentElement.insertBefore scriptEl, document.documentElement.firstElementChild
```

Comment:
`req.open(,,false)` will cause a warning, which is a little ugly.

You said in a comment in #1405 :
> CSP stops us from injecting our hook before page scripts are done loading.

What prevent a `<script>` with `[src]` from executing is its file's origin (host). Once I found an explanation:
* a `<script src="/...">` will run immediately, and `$0.async` is `false`
* a `<script src="http://other-site/...">` will be delayed, and two such scripts will run without any order after page is loaded.

So, I think here we can add `scriptEl.async = false` before `insertBefore` to make this script run at once.
``` coffeescript
for script in injectScripts
  ...
  scriptEl.src = chrome.runtime.getURL(script)
  scriptEl.async = false
  document....
```

And a simple test passed: